### ISSUE TITLE
Uchiwa 0.11.0 and above does not have a /get_sensu endpoint

### ISSUE THREAD

**Issue by rokkamokka** [warmfusion/mosaic#7]
Uchiwa removed the /get_sensu endpoint back in 0.11.0 (latest version as of now is 0.18.1).

**Comment by warmfusion** [comment]
Confirmed. I was thinking of implementing against the sensu API directly rather than using uchiwa as a bridge, but that would mean exposing that API publicly. Any opinions?

**Comment by rokkamokka** [comment]
Either would work fine for our use case. We host the dashboards locally on the sensu server anyway. Implementing directly against the sensu API removes one dependency at least, which is nice (for those that want to run mosaic but not uchiwa).

**Comment by warmfusion** [comment]
Agreed. Given the read only nature of the interactions I'd recommend anyone wanting to expose the API to restrict to GET requests only. 

Ok, I'm not sure when I'll have time to make the changes, but it'll point at sensu-api rather than uchiwa when it's done.

**Comment by warmfusion** [comment]
I've spoken with some colleagues about a reimplementation using React rather than AngularJS for the front-end.

I'll investigate this as an option - I'm not a JS developer so it might take a little while.
